service/raft/raft_group0.cc

```diff
@@ -971,32 +1002,29 @@ static future<> wait_until_every_peer_joined_group0(db::system_keyspace& sys_ks,
         std::set_difference(peers.begin(), peers.end(), current_config.begin(), current_config.end(), std::back_inserter(missing_peers));
 
         if (missing_peers.empty()) {
-            if (!group0_config.is_joint()) {
+            if (!members0.is_joint()) {
                 co_return;
             }
 
-            upgrade_log.info("group 0 configuration is joint: {}. Sleeping for a while before retrying...", group0_config);
-            co_await sleep_abortable(retry_period, as);
+            upgrade_log.info("group 0 configuration is joint: {}. Sleeping for a while before retrying...", current_config);
```

Comment:
Unfortunately the message is less useful now, previously it would print the joint configuration in detail (both the current and previous configs) so the user would know what exactly the config change is. That was the purpose of the message:
"group 0 configuration is joint: (here the joint config is printed)"
now it will just print a single set of IPs.

Maybe you could still obtain the config from `members0` and print it...

Or maybe it doesn't matter that much.

Reply:
Yes, I figured it doesn't matter much.